Sources/PackageConfig/Package.swift

```diff
@@ -100,12 +100,35 @@ enum Package {
 		let fileManager = FileManager.default
 
 		let swiftPMDir = swiftPath.replacingOccurrences(of: "bin/swiftc", with: "lib/swift/pm")
-		let versions = try! fileManager.contentsOfDirectory(atPath: swiftPMDir)
-		#warning("TODO: handle //swift-tools-version:4.2 declarations")
-		let latestSPM = versions.sorted().last!
-		let libraryPathSPM = swiftPMDir + "/" + latestSPM
+		let versions = try! fileManager.contentsOfDirectory(atPath: swiftPMDir).filter { $0 != "llbuild" }
+
+        let swiftToolsVersion = getSwiftToolsVersion()
```

Comment:
I think this code probably needs to be re indented

Reply:
Probably we should use https://github.com/shibapm/Komondor to be sure everything is formatted correctly 🤔